docs/pages/application-access/guides/azure.mdx

```diff
@@ -0,0 +1,554 @@
+
+Once creation finishes, click **Go to resource**. On the page for the new
+identity, click **JSON View**. At the top of the right sidebar, you will see a
+field called **Resource ID** with a value resmmbling the following:
+
+```
+/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/my-resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/teleport-azure
+```
+
+Copy this and paste it somewhere safe. We will use this later in the
+guide.
+
+### Allow the `teleport-azure` identity to view resources
+
+Once you create an Azure identity, authorize it to access resources in your
+account. In this case, we will authorize your new Azure identity to view
+resources in its resource group.
+
+Enter the name of your Azure resource group in the Azure Portal search box and
+visit the page for that resource group. On the left navigation sidebar, click
+the **Access control (IAM)** tab. In the row of buttons at the top of the
+**Access control (IAM)** panel, click **Add > Add role assignment**.
+
+Within the **Add role assignment** screen, click **Reader**, a built-in role
+with view-only access to resources. 
+
+![Add a role
+assignment](../../../img/application-access/azure/add-role-assignment.png)
+
+Scroll to the bottom of the screen and click **Next**.
+
+Within the **Members** tab, in the **Assign access to** field, choose **Managed
+identity**. Click **Select members**.
+
+On the right sidebar, find the **Managed identity** dropdown menu and select
+**User-assigned managed identity**. Choose the `teleport-azure` identity you
+created earlier. 
+
+![Select managed
+identities](../../../img/application-access/azure/select-managed-identities.png)
+
+Click **Select**, then **Review + assign**.
+
+Verify that your **Role** is "Reader", the **Scope** matches your chosen
+resource group, and the **Members** field includes the `teleport-azure` managed
+identity you created earlier.
+
+Click **Review + assign** again.
+
+### Attach an identity to your Azure VM
+
+Now that you have created a managed identity and assigned it a role, attach the
+identity to your Azure VM so the Teleport Application Service can assume the
+identity in order to proxy Azure CLI traffic.
+
+In the [Virtual machines
+view](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines)
+of Azure Portal, click on the name of the VM you are using to host the Teleport
+Application Service. 
+
+On the right side panel, click the **Identity** tab, then within the
+**Identity** view, click the **User assigned** tab. Click **+Add**, then select
+the `teleport-azure` identity. Click **Add**.
+
+![Add an identity to a
+VM](../../../img/application-access/azure/vm-identity.png)
+
+## Step 2/4. Deploy the Teleport Application Service
+
+In this step, you will run the Teleport Application Service on the Azure VM you
+assigned the `teleport-azure` identity to. The Teleport Application Service
```

Comment:
Since this section is about running the Application Service, and I've added more context about how the Application Service works at the beginning of the guide, I've suggested removing everything in this paragraph after the first sentence.